apollo-router/src/plugins/connectors/testdata/nullability.yaml

```
@@ -56,4 +56,18 @@ subgraphs:
               """
               entity: true
             )
+
+          defaultArgs(
+            str: String = "default"
+            int: Int = 42
+            float: Float = 1.23
+            bool: Boolean = true
+            # TODO: enums after 2.10.0-alpha.1
```

```suggestion
            # TODO: enums after 2.10.0-alpha.2
```

(alpha 1 exists, that was the fbp fix)